src/app/modules/time-entries/pages/time-entries.component.spec.ts

```diff
@@ -375,19 +377,59 @@ describe('TimeEntriesComponent', () => {
     component.entry = null;
     component.entryId = null;
     const lastEntry = {
-      description : 'testing is fun',
-      technologies : [],
-      uri : 'http://testing.is.fun',
-      activity_id : 'sss',
-      project_id : 'id',
-      start_date : new Date(new Date().setHours(0, 0, 0, 0)),
-      end_date : new Date(new Date().setHours(0, 0, 0, 0))
+      description: 'testing is fun',
+      technologies: [],
+      uri: 'http://testing.is.fun',
+      activity_id: 'sss',
+      project_id: 'id',
+      start_date: new Date(new Date().setHours(0, 0, 0, 0)),
+      end_date: new Date(new Date().setHours(0, 0, 0, 0))
     };
-    state.timeEntriesDataSource.data = [ lastEntry ];
+    state.timeEntriesDataSource.data = [lastEntry];
     mockEntriesSelector = store.overrideSelector(getTimeEntriesDataSource, state.timeEntriesDataSource);
 
-    component.projectSelected({ projectId : 'id'});
+    component.projectSelected({ projectId: 'id' });
     expect(component.entry).toEqual(lastEntry);
   }));
 
+  it('when the data source is loaded, the table should to show the appropriated column titles', waitForAsync(() => {
+    component.timeEntriesDataSource$.subscribe(() => {
+
+      fixture.detectChanges();
+
+      const expectedColumnTitles = [
+        'Date',
+        'Time in - out',
+        'Duration',
+        'Customer',
+        'Project',
+        'Activity',
+        '',
```

Comment:
@israteneda  why is this empty? `''`

Reply:
It's because in this last column there is no column title but in the next rows, buttons are presented.
![image](https://user-images.githubusercontent.com/20668624/105726799-73a49b80-5ef8-11eb-9e6c-fb69d7fc4dd6.png)